### DISCUSSION TITLE
When should I use the "generate bin files" command?

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#449] [Q&A] [answered]
When should I use the "generate bin files" command?
![binfiles](https://user-images.githubusercontent.com/3830326/134588835-03735116-22fc-465e-a3b2-3e07cacc406c.png)

**Comment by johnlindquist** [comment, marked as answer]
Rarely, if ever. 

The app regenerates the `~/.kenv/bin` directory on startup. Early on the project, that didn't happen and people were installing scripts and needed to generate bins. I'm also considering allowing people to customize the bin template, but probably not anytime soon.
